### ISSUE TITLE
PayJoin support

### ISSUE THREAD

**Issue by Kixunil** [btcontract/lnwallet#33]
It'd be great if BLW supported PayJoin. Apart from obvious privacy benefit, there's an interesting use case: consolidation of inputs.

Suppose a channel was closed with somewhat small value of UTXO. The current process of consolidating it needs one transaction to move it to a wallet with other UTXOs and a second consolidation transaction leading to quite high fees. PayJoin would enable doing it in a single step.

**Comment by btcontract** [comment]
In general my views are such that any technique which claims that privacy enhancement on L1 is its main objective is overall harmful and should be avoided, so not a fan of any join at all.

As to UTXO consolidation, do you mean some transaction here which gets constructed across multiple wallets?

**Comment by Kixunil** [comment]
Yes, PayJoin enables creating a transaction that joins UTXOs from two wallets in one transaction.

**Comment by btcontract** [comment]
OK, thanks for letting me know, I'll look into it once I have more free time. Consolidation option makes this somewhat interesting.
